index.html

```diff
@@ -65,7 +65,7 @@ <h2>Projects</h2>
       <article class="project">
         <img class="project-1-img" src="images/project-img-4.png" alt="project 1imgage">
         <section class="rectangle">
-          <p>Project Tinder clone</p>
+          <p>Project Tinders clone</p>
           <ul class="technology-tag">
```

Comment:
- [ ] Kindly add the absolute link for your image to avoid any future concerns when image is not in the same repos